internal/cmd/resolve/resolve.go

```diff
@@ -49,7 +54,18 @@ Example:
 $ debricked resolve . `+exampleFlags)
 	cmd.Flags().BoolVar(&verbose, VerboseFlag, true, "set to false to disable extensive resolution error messages")
 
+	npmPreferredDoc := strings.Join(
+		[]string{
+			"This flag allows you to select which package manager will be used as a resolver: Yarn (default) or NPM.",
+			"Examples:",
+			"$ debricked resolve --prefer-npm",
+			"$ debricked resolve -n",
```

Comment:
not sure we should have this shorthand flag, this flag feels way too uncommon to have a shorthand like that

Reply:
Do we need a shorthand for this command?